apps/AEPSampleApp/extensions/Core.js

```diff
@@ -1,6 +1,6 @@
 import React, {Component} from 'react';
 import {Button, Platform, StyleSheet, Text, View, ScrollView, NativeModules} from 'react-native';
-import {AEPCore, AEPLifecycle, AEPSignal, AEPMobileLogLevel, AEPMobilePrivacyStatus, AEPMobileVisitorAuthenticationState, AEPVisitorID, AEPExtensionEvent} from '@adobe/react-native-aepcore';
+import {AEPIdentity, AEPCore, AEPLifecycle, AEPSignal, AEPMobileLogLevel, AEPMobilePrivacyStatus, AEPMobileVisitorAuthenticationState, AEPVisitorID, AEPExtensionEvent } from '@adobe/react-native-aepcore';
```

Comment:
https://github.com/adobe/aepsdk-react-native/blob/staging/apps/AEPSampleApp/extensions/Identity.js
Above is the separate view/screen for Ientity. Can we remove the Identity sample code from Core.js?